### PR TITLE
Proofread and corrected annotations.md and aop.md.

### PR DESCRIPTION
**Pull request by smolinari**
Please make sure the term "Aspects" I added fits correctly to the whole concept of AOP within appserver. I thought the term was needed to help explain AOP and the need for the rest of the terms.

Scott

### CODE REVIEW

File: _docs/annotations.md [hunk 1]
```diff
@@ -2,20 +2,20 @@
...
+meta_description: Since one of our main goals is to make configuration as simple as possible, we decided to use annotations wherever possible.
 position: 60
 group: Docs
 permalink: /get-started/documentation/annotations.html
 ---
 
-As one of our main targets is making the configuration as simple as possible, we have decided to use
-annotations wherever possible. Annotations are not supported natively by PHP. Therefore, we provide
+Since one of our main goals is making the configuration of appserver as simple as possible, we have decided to use
+annotations wherever we cann. Annotations are not supported natively by PHP. Therefore, we provide
```
Comment (wick-ed) [inline _docs/annotations.md]: cann !== can?

File: _docs/aop.md [hunk 2]
```diff
@@ -112,36 +114,43 @@ The classloader configuration might look like the following.
...
- We currently support two join points, one standalone and the other one as addition for other join points.
- 
+
+ We currently support two join point types.
+
 | Keyword      | Type       | Description                                                     |
 | -------------| -----------| ----------------------------------------------------------------|
 | `call`       | standalone | The call of a method being static or dynamic.                   |
-| `execute`    | addition   | An addition to other join points. Describes that a certain join point has to be in the context of an execution stack. E.g. a call to `methodB` might only be interesting if it was made from within `methodA` |
+| `execute`    | addition   | An addition to other join points. Describes that a certain join point has to be in the context of an execution stack. E.g. a call to `methodB` will only be carried out, if it was made from within `methodA` |
```
Comment (wick-ed) [inline _docs/aop.md]: Is "carried out" (vornehmen) correct? The call will be made anyway, but the join point would not match if the call stack is anything else than methodA -> methodB

File: _docs/aop.md [hunk 3]
```diff
@@ -248,15 +258,15 @@ class AnExampleAspect
 
 Aspect classes have to be implemented within your webapp, namely within the `META-INF`, `WEB-INF` or `common` directory.
 
-> Please note that aspects MUST NOT have a constructor requiring parameters.
+> Please note that aspects CANNOT contain a constructor requiring parameters.
```
Comment (wick-ed) [inline _docs/aop.md]: Has to stay `MUST NOT` as of [RFC 2119](http://www.ietf.org/rfc/rfc2119.txt)

File: _docs/aop.md [hunk 4]
```diff
@@ -2,48 +2,49 @@
...
 
-We do not want to force a certain way in which objects are instantiated. We have to ensure that the proxy and the original implementation can be swapped externally, without using a factory or entity manager.
-Therefore, we are working with the original definition of a structure (class, trait, or in another context interface).
+The configuration's details are described further below. For now, it is important to know that there are alternative classes that wrap around the original classes and execute cross-cutting logic.
 
-As we still want to switch between original and proxy we simply generate another implementation of the original class, we call it `ExampleClass`. It contains method stubs that wrap around the original methods. As proxy generation is a very expensive thing, we will store these new definitions in the file system for later re-use.
+We don't want to do is force a certain way to instantiate objects. We have to ensure that the proxy and the original implementation can be swapped externally, without using a factory or entity manager. Therefore, we are working with the original definition of a structure (class, trait, or in another context interface).
```
Comment (wick-ed) [inline _docs/aop.md]: "What we don't want to do"?

**Comment by wick-ed** [comment]
Hi Scott,

I made some inline comments and would be glad if you could make another commit to address them.
Thanks for your contribution! :+1:

File: _docs/annotations.md (hunk 1 above)
Comment (smolinari) [inline _docs/annotations.md]: hehehe... correction to the corrections.

File: _docs/aop.md (hunk 2 above)
Comment (smolinari) [inline _docs/aop.md]: Hmmm...would this be better?

E.g. when a call to `methodB` is made from within `methodA`, then the execute join point is a match and would also be carried out.

Comment (wick-ed) [inline _docs/aop.md]: Make it "and woven advice will be carried out" and I am happy ^^

File: _docs/aop.md (hunk 3 above)
Comment (smolinari) [inline _docs/aop.md]: Ok. How is this?

Please note that the constructor of aspect classes MUST NOT require parameters.

Comment (wick-ed) [inline _docs/aop.md]: Great :)

**Comment by smolinari** [comment]
I'll make the changes this evening, when I get home. Was the addition of the "Aspects" term ok?

Scott

**Comment by wick-ed** [comment]
You mean the prism metaphor? I think that makes the term more clear, yes

File: _docs/aop.md (hunk 2 above)
Comment (smolinari) [inline _docs/aop.md]: Final sentence. 

E.g. when a call to methodB is made from within methodA, then the execute join point is a match and the woven advice will be carried out.

**Comment by smolinari** [comment]
Supi. I am learning and I have found out the best way to learn is to help support what it is you want to learn, however you can.

File: _docs/aop.md (hunk 2 above)
Comment (smolinari) [inline _docs/aop.md]: E.g. when a call to methodB is made from within methodA, then the execute join point is a match and the woven advice will be carried out.

**Pull request merged**